udffsck/main.c

```diff
@@ -71,7 +71,18 @@ void user_interrupt(int dummy) {
  * It instructs user to report it, because this behavior is bug.
  */
 void segv_interrupt(int dummy) {
-    fatal("Unexpected error, please report it. More info at man page. Exiting.\n");
+    fatal("Unexpected error (SEGV), please report it. More info at man page. Exiting.\n");
+    exit(8);
+}
+
+/**
+ * \brief Bus error handler
+ *
+ * If program dies because of SIGBUS, this handler catches it and die properly.
+ * It instructs user to report it, because this behavior is bug.
+ */
+void sigbus_interrupt(int dummy) {
+    fatal("Unexpected error (SIGBUS), please report it. More info at man page. Exiting.\n");
```

Comment:
It is normal to get SIGBUS for applications which use mmap. If somebody call truncate(0) on file then application which mmaped that file get SIGBUS.

Reply:
Ok, that is true. 

Either way, program is supposed to die, so I should change message to something more meaningful, right?
My suggestion would be something like: Medium changed its size during run, is somebody manipulating with it?

Reply:
Anyway, updated message is here:  c32a0e9

Reply:
Ok.